lib/active_admin_import/model.rb

```diff
@@ -8,7 +8,7 @@ class Model
     include ActiveModel::Validations::Callbacks
 
     validates :file, presence: {
-      message: proc { I18n.t("active_admin_import.no_file_error") }
+        message: proc { I18n.t("active_admin_import.no_file_error") }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.